jobs/cf-kibana/templates/kibana.yml.erb

```diff
@@ -57,12 +59,14 @@ kibana.index: ".kibana"
 # Time in milliseconds to wait for responses from the back end or Elasticsearch. This value
 # must be a positive integer.
 # elasticsearch.requestTimeout: 300000
+elasticsearch.requestTimeout: <%= p('cf-kibana.request_timeout') %>
 
 # Time in milliseconds for Elasticsearch to wait for responses from shards. Set to 0 to disable.
 # elasticsearch.shardTimeout: 0
 
 # Time in milliseconds to wait for Elasticsearch at Kibana startup before retrying.
 # elasticsearch.startupTimeout: 5000
+elasticsearch.shardTimeout: <%= p('cf-kibana.shard_timeout') %>
```

Comment:
@jmcarp could you move this parameter to the line 66 for the better readability of the kibana config. Otherwise, looks good, thanks!